lms/djangoapps/onboarding/models.py

```diff
@@ -487,6 +497,17 @@ class UserExtendedProfile(TimeStampedModel):
 
     history = HistoricalRecords()
 
+    # Onboarding model optimization - new fields
+    hear_about_philanthropyu = MultiSelectWithOtherField(choices=choices.HEAR_ABOUT_PHILANTHROPY_LABELS,
+                                                         other_max_length=255,
+                                                         max_choices=1,
+                                                         null=True,
+                                                         blank=True)
+    function_areas = MultiSelectField(choices=choices.FUNCTIONS_LABELS, null=True, blank=True)
+    interests = MultiSelectField(choices=choices.INTERESTS_LABELS, null=True, blank=True)
+    learners_related = MultiSelectField(choices=choices.INTERESTED_LEARNERS_LABELS, null=True, blank=True)
+    goals = MultiSelectField(choices=choices.GOALS_LABELS, null=True, blank=True)
```

Comment:
Let's please add `TODO: ...` on all fields which are redundant after addition of these fields.